### PR TITLE
Refactoring of API s3

### PR DESCRIPTION
**Pull request by flicker-harmony**
## Issue

Some refactoring of current APIs on s3.

### Unit Test Coverage

Before:

```
<!-- copy/paste 'go test -cover' result in the directory you made change -->
```

After:

```
<!-- copy/paste 'go test -cover' result in the directory you made change -->
```

## Operational Checklist

1. **Does this PR introduce backward-incompatible changes to the on-disk data structure and/or the over-the-wire protocol?**. (If no, skip to question 8.)

    **YES|NO**

2. **Describe the migration plan.**. For each flag epoch, describe what changes take place at the flag epoch, the anticipated interactions between upgraded/non-upgraded nodes, and any special operational considerations for the migration.

3. **Describe how the plan was tested.**

4. **How much minimum baking period after the last flag epoch should we allow on Pangaea before promotion onto mainnet?**

5. **What are the planned flag epoch numbers and their ETAs on Pangaea?**

6. **What are the planned flag epoch numbers and their ETAs on mainnet?**

    Note that this must be enough to cover baking period on Pangaea.

7. **What should node operators know about this planned change?**

8. **Does this PR introduce backward-incompatible changes *NOT* related to on-disk data structure and/or over-the-wire protocol?** (If no, continue to question 11.)

    **YES|NO**

9. **Does the existing `node.sh` continue to work with this change?**

10. **What should node operators know about this change?**

11. **Does this PR introduce significant changes to the operational requirements of the node software, such as >20% increase in CPU, memory, and/or disk usage?**

### CODE REVIEW

File: internal/hmyapi/backend.go [hunk 1]
```diff
@@ -112,31 +112,31 @@ func GetAPIs(b Backend) []rpc.API {
...
 		{
-			Namespace: "hmyv2",
+			Namespace: "hmy_v2",
 			Version:   "1.0",
 			Service:   apiv2.NewPublicHarmonyAPI(b),
 			Public:    true,
 		},
 		{
-			Namespace: "hmyv2",
+			Namespace: "hmy_v2",
```
Comment (fxfactorial) [inline internal/hmyapi/backend.go]: Same question like on other PR, will it break trustwallet or anyone

Comment (flicker-harmony) [reply internal/hmyapi/backend.go]: Asked

Comment (fxfactorial) [reply internal/hmyapi/backend.go]: excellent, just read in tg. looks good, we should merge .

**Comment by LeoHChen** [comment]
@flicker-harmony, There are three things I'd request for you as the owner of all the RPC APIs.

1) Please make sure s3 and master branch matches in terms of the API.
2) To prevent API regression, please open a ticket to Daniel to add hmy_v2 API test in Jenkins job.
3) All the new APIs added into s3 or master branch, please make sure we have it properly documented, postman added, and test case coverage.

**Pull request merged**